src/lightning_app/cli/lightning_cli_delete.py

```diff
@@ -40,6 +47,164 @@ def delete_cluster(cluster: str, force: bool = False, do_async: bool = False) ->
     cluster_manager.delete(cluster_id=cluster, force=force, do_async=do_async)
 
 
+def _find_cluster_for_user(app_name: str, cluster_id: Optional[str]) -> str:
+    console = Console()
+    cluster_manager = AWSClusterManager()
+
+    default_cluster, valid_cluster_list = None, []
+    for cluster in cluster_manager.list_clusters():
+        valid_cluster_list.append(cluster.id)
```

Comment:
is the list somehow sorted that you can break with first found?

Reply:
it is not. in the event a user did not enter a valid cluster ID we need to prompt them with the full list of clusters in the function below :)